app/Resources/views/admin/index.html.twig

```diff
@@ -126,7 +129,7 @@
                 </a>
             {% endif %}
 
-            {% if is_granted("ROLE_FINANCE_MANAGER") %}
+            {% if is_granted("ROLE_FINANCE_MANAGER") and not oidc_enable %}
```

Comment:
Est-ce que ca ne serait pas plutôt jolie de dire que certain role ne sont pas accessible avec keycloak ? Ca éviterait de diminuerait le nombre de `not oidc_enable` ?

Reply:
En particulier, faudrait pas que dans les futurs évolutions futures de l'espace membre, on fasse des choses non compatible avec keycloack.

Reply:
Oui, mais je vois pas comment tu peux faire ? sachant que tu peux activer / désactiver open id alors que ta base est déjà peuplée et les user avec déjà des droits ?
Une idée d'où te brancher pour gérer ça ?

Pour la dépendance à keycloak, je vois pas trop comment tu pourrais l'éviter. A partir du moment ou tu ajoutes une fonctionnalité, tu es obligé d'en tenir compte dans tous les nouveaux développements non ? tu pense à un moyen de la rendre vraiment transparente pour la suite ?

Reply:
> sachant que tu peux activer / désactiver open id alors que ta base est déjà peuplée et les user avec déjà des droits ?

Effectivement je n'avais pas pensé à ce cas d'usage.


Pour la dépendance, pas simple. Pour le moment on prenait l'espace membre comme une appli monolithique. Faudrait être essayer d'y réfléchir plus sous forme de module dans la conception ?